src/Sonarr.Api.V3/Episodes/RenameEpisodeController.cs

```diff
@@ -26,5 +27,17 @@ public List<RenameEpisodeResource> GetEpisodes(int seriesId, int? seasonNumber)
 
             return _renameEpisodeFileService.GetRenamePreviews(seriesId).ToResource();
         }
+
+        [HttpGet("multiple")]
+        [Produces("application/json")]
+        public List<RenameEpisodeResource> GetEpisodes(List<int> seriesIds)
+        {
+            if (seriesIds is not { Count: not 0 })
```

Comment:
Lot of negatives here, but looks like this can be simplified to
```suggestion
            if (seriesIds is { Count: 0 })
```